src/NzbDrone.Core/Download/Pending/PendingReleaseService.cs

```diff
@@ -189,6 +189,7 @@ public List<Queue.Queue> GetPendingQueue()
                         Movie = pendingRelease.RemoteMovie.Movie,
                         Quality = pendingRelease.RemoteMovie.ParsedMovieInfo?.Quality ?? new QualityModel(),
                         Languages = pendingRelease.RemoteMovie.ParsedMovieInfo?.Languages ?? new List<Language>(),
+                        Year = pendingRelease.ParsedMovieInfo?.Year,
```

Comment:
This is using the parsed year and not the matched movie's year, I assume this was intentional since the matched movie's year is already available under the `Movie` prop?

Reply:
I used the movie prop in FE. Thanks.